perma_web/perma/views/user_management.py

```diff
@@ -985,6 +985,9 @@ def settings_organizations(request):
     if pending_registrar:
         messages.add_message(request, messages.INFO, "Thank you for requesting an account for your library. Perma.cc will review your request as soon as possible.")
 
+    organizations = request.user.organizations.all().order_by('registrar')
+    orgs_by_registrar = {registrar : [org for org in orgs] for registrar, orgs in itertools.groupby(organizations, lambda x: x.registrar)}
```

Comment:
Actually: nope. That does not work. Happy to show you on my machine if you like: I've altered a test user to have lots of affiliations, which is where problems show up.